selfservice/strategy/saml/provider_config.go

```diff
@@ -0,0 +1,34 @@
+package saml
+
+type Configuration struct {
+	// ID is the provider's ID
+	ID string `json:"id"`
+
+	// Label represents an optional label which can be used in the UI generation.
+	Label string `json:"label"`
+
+	// Represent the path of the certificate of your application
```

Comment:
Standard Golang comment format is // <variable name> <description>

```suggestion
	// PublicCertPath path of your application certificate
```

Same for other variables below